bootstrap/Makefile

```diff
@@ -112,8 +112,8 @@ build-gc: deepcopy generate fmt vet
 build-kfctl-tgz: build-kfctl
 	chmod a+rx ./bin/kfctl
 	rm -f bin/*.tgz
-	cd bin && tar -cvzf kfctl_$(TAG)_linux.tar.gz ./kfctl
-	cd bin && tar -cvzf kfctl_${TAG}_darwin.tar.gz ./kfctl-darwin
+	cd bin && tar -cvzf kfctl_$(TAG)_linux.tar.gz ./kfctl && rm ./kfctl
```

Comment:
Would it be better to use different subdirectories for different architectures?
e.g
bin/${ARCH}/kfctl

Swapping out bin/kfctl for different architectures seems like it could lead to some confusing error

Reply:
Oh yeah much better 😅